src/api/createVoiceVideoCalling.ts

```diff
@@ -340,11 +340,26 @@ export class VoiceVideoCallingProxy {
                 SecondaryChannelEventType: SecondaryChannelEvents.End
             }
 
+            this.scenarioMarker?.startScenario(TelemetryEvent.OnCallDisconnected, {
+                CallId: this.callId || ''
+            });
+
             try {
                 this.callingParams?.OCClient.makeSecondaryChannelEventRequest(this.callingParams?.chatToken.requestId, body);
                 this.debug && console.debug(`[VoiceVideoCallingProxy][onCallDisconnected][makeSecondaryChannelEventRequest]`);
+                this.scenarioMarker?.completeScenario(TelemetryEvent.OnCallDisconnected,{
+                    CallId: this.callId || ''
+                });
             } catch (e) {
                 console.error(`[VoiceVideoCallingProxy][onCallDisconnected][makeSecondaryChannelEventRequest] Failure ${e}`);
+                
+                const exceptionDetails = {
+                    response: 'OnCallDisconnectedFailed'
```

Comment:
It's the OCClient.makeSecondaryChannelEventRequest that's failing. So `response` should be `OCClientMakeSecondaryChannelEventRequestFailed`